new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -518,4 +564,85 @@ public static boolean deployApplication(String appName, String appLocation, Stri
                                           String username, String password, String target) {
     return true;
   }
+
+  // ------------------------ some utility method  -------------------------
+
+  /**
+   * Check running some command and returning expected value.
+   *
+   * @param command the command to run
+   * @param expectedValue the expected value should return
+   * @return true on success, false otherwise
+   */
+  public static boolean getExpectedResult(String command, String expectedValue) {
+    logger.info("Running command - \n" + command);
+
+    ExecResult result;
+    try {
+      result = ExecCommand.exec(command);
+      if (result.exitValue() != 0) {
+        logger.info("Command failed with errors " + result.stderr() + "\n" + result.stdout());
+        return false;
+      }
+    } catch (Exception e) {
+      logger.info("Got exception, command failed with errors " + e.getMessage());
+      return false;
+    }
+
+    if (!result.stdout().contains(expectedValue)) {
```

Comment:
this method is not checking that the command returned the expected value - it is checking the the stdout from the command contained some specified text.  rename it, update doc to be accurate.